backend/api/serializers/CreditTradeComment.py

```diff
@@ -50,7 +50,16 @@ class Meta:
         model = CreditTradeComment
         fields = (
             'id', 'credit_trade', 'comment', 'privileged_access', 'create_timestamp',
-            'update_timestamp', 'create_user')
+            'update_timestamp', 'create_user', 'update_user')
+
+        read_only_fields = ('id', 'create_timestamp', 'create_user', 'credit_trade',
+                            'privileged_access')
+
+
+class CreditTradeCommentCreateSerializer(serializers.ModelSerializer):
+    class Meta:
+        model = CreditTradeComment
+        fields = ('id', 'credit_trade', 'comment', 'privileged_access', 'create_user', 'update_user')
+        read_only_fields = ['id']
```

Comment:
Shouldn't this be a tuple?

Reply:
It can be either a list or a tuple, but I'll make it a tuple for consistency with the others.